src/lib/core/CHIPTLVDebug.cpp

```diff
@@ -68,13 +68,11 @@ static void DumpHandler(DumpWriter aWriter, const char * aIndent, const TLVReade
     temp.Init(aReader);
     tagControl = static_cast<TLVTagControl>(temp.GetControlByte() & kTLVTagControlMask);
 
-    aWriter("%zd ", aDepth);
+    aWriter("0x%02X, ", temp.GetLengthRead());
```

Comment:
Note for reviewers (@tcarmelveilleux  @mrjerryjohns ). 

The outputs below are for the following command:
```
./out/debug/standalone/chip-cert print-cd credentials/development/commissioner_dut/struct_cd_authorized_paa_list_count10_valid/cd.der
```

The printing format prior to my changes in this file were:
```
0 0x7ffd0af72841, tag[Anonymous]: 0xffffffff, type: Structure (0x15), container: 
1     0x7ffd0af72844, tag[Context Specific]: 0x0, type: Signed Fixed Point (0x00), value: 1
1     0x7ffd0af72848, tag[Context Specific]: 0x1, type: Unsigned Fixed Point (0x04), value: 65521
1     0x7ffd0af7284a, tag[Context Specific]: 0x2, type: Array (0x16), container: 
2         0x7ffd0af7284d, tag[Anonymous]: 0xffffffff, type: Unsigned Fixed Point (0x04), value: 32768
1     0x7ffd0af72852, tag[Context Specific]: 0x3, type: Unsigned Fixed Point (0x04), value: 4660
1     0x7ffd0af72855, tag[Context Specific]: 0x4, type: UTF-8 String (0x0c), length: 19, value: "ZIG20141ZB330001-24"
1     0x7ffd0af7286b, tag[Context Specific]: 0x5, type: Unsigned Fixed Point (0x04), value: 0
1     0x7ffd0af7286e, tag[Context Specific]: 0x6, type: Unsigned Fixed Point (0x04), value: 0
1     0x7ffd0af72872, tag[Context Specific]: 0x7, type: Unsigned Fixed Point (0x04), value: 39030
1     0x7ffd0af72875, tag[Context Specific]: 0x8, type: Unsigned Fixed Point (0x04), value: 0
1     0x7ffd0af72877, tag[Context Specific]: 0xb, type: Array (0x16), container: 
2         0x7ffd0af72879, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af72879

2         0x7ffd0af7288f, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af7288f

2         0x7ffd0af728a5, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af728a5

2         0x7ffd0af728bb, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af728bb

2         0x7ffd0af728d1, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af728d1

2         0x7ffd0af728e7, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af728e7

2         0x7ffd0af728fd, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af728fd

2         0x7ffd0af72913, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af72913

2         0x7ffd0af72929, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af72929

2         0x7ffd0af7293f, tag[Anonymous]: 0xffffffff, type: Data (0x10), length: 20, value: 0x7ffd0af7293f
```

The printing format after my updates in this file is:
```
0x01, tag[Anonymous]: 0xffffffff, type: Structure (0x15), container: 
0x04,     tag[Context Specific]: 0x0, type: Signed Fixed Point (0x00), value: 1
0x08,     tag[Context Specific]: 0x1, type: Unsigned Fixed Point (0x04), value: 65521
0x0A,     tag[Context Specific]: 0x2, type: Array (0x16), container: 
0x0D,         tag[Anonymous]: 0xffffffff, type: Unsigned Fixed Point (0x04), value: 32768
0x12,     tag[Context Specific]: 0x3, type: Unsigned Fixed Point (0x04), value: 4660
0x15,     tag[Context Specific]: 0x4, type: UTF-8 String (0x0c), length: 19, value: "ZIG20141ZB330001-24"
0x2B,     tag[Context Specific]: 0x5, type: Unsigned Fixed Point (0x04), value: 0
0x2E,     tag[Context Specific]: 0x6, type: Unsigned Fixed Point (0x04), value: 0
0x32,     tag[Context Specific]: 0x7, type: Unsigned Fixed Point (0x04), value: 39030
0x35,     tag[Context Specific]: 0x8, type: Unsigned Fixed Point (0x04), value: 0
0x37,     tag[Context Specific]: 0xb, type: Array (0x16), container: 
0x39,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:6AFD22771F511FECBF1641976710DCDC31A1717E
0x4F,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BBCABBC5016577AA8B44FFB90FCCA140FE6620
0x65,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB35BBC5016577AA8B44FFB90FCCA140FE6620
0x7B,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB3544C5016577AA8B44FFB90FCCA140FE6620
0x91,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB35443A016577AA8B44FFB90FCCA140FE6620
0xA7,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB35443AFE6577AA8B44FFB90FCCA140FE6620
0xBD,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB35443AFE9A77AA8B44FFB90FCCA140FE6620
0xD3,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB35443AFE9A88AA8B44FFB90FCCA140FE6620
0xE9,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB35443AFE9A88558B44FFB90FCCA140FE6620
0xFF,         tag[Anonymous]: 0xffffffff, type: Octet String (0x10), length: 20, value: hex:F4BB35443AFE9A88557444FFB90FCCA140FE6620
```